### DISCUSSION TITLE
System.Text.Json: Ignore Invalid Enum

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/runtime#46003] [Q&A]
Does anyone have code for an enum converter that tries to deserialize a value and if it doesn't, returns a default value (or perhaps null) instead?

**Comment by BjornDeRijcke** [comment]
This might help: https://stackoverflow.com/questions/57893720/how-can-i-get-a-null-value-instead-of-a-serialization-error-when-deserializing-a
